.github/workflows/release.yml

```
@@ -0,0 +1,115 @@
+name: Create Release
+on:
+    workflow_dispatch:
+    schedule:
+        - cron: '0 9 * * 1'
+jobs:
+    version:
+        name: Get versions and meta
+        timeout-minutes: 5
+        runs-on: ubuntu-latest
+        outputs:
+            LATEST_TAG_NAME: ${{ steps.getSimpleIconsVersion.outputs.latest_tag_name }}
+            LATEST_RELEASE_NAME: ${{ steps.getSimpleIconsVersion.outputs.latest_release_name }}
+            PREVIOUS_VERSION: ${{ steps.getSimpleIconsVersion.outputs.previous_version }}
+        steps:
+            - name: Checkout Repo
+              uses: actions/checkout@v3
+
+            - name: Get Simple Icons version and release name plus previous version of icon pack
+              id: getSimpleIconsVersion
+              run: |
+                simple_icons_release_data="$(curl --retry 5 -s https://api.github.com/repos/simple-icons/simple-icons/releases/latest)"
+                echo "latest_tag_name=$(echo "$simple_icons_release_data" | jq -r '.tag_name')" >> $GITHUB_OUTPUT
+                echo "latest_release_name=$(echo "$simple_icons_release_data" | jq -r '.name')" >> $GITHUB_OUTPUT
+                echo "previous_version=$(jq -r '.Version' template/manifest.json)" >> $GITHUB_OUTPUT
+
+    build:
+        name: Build Icon Pack
+        timeout-minutes: 10
+        needs: version
+        if: ${{ needs.version.outputs.LATEST_TAG_NAME != needs.version.outputs.PREVIOUS_VERSION }}
+        runs-on: windows-latest
+        permissions: write-all
+        steps:
+            - name: Checkout Repo
+              uses: actions/checkout@v3
+
+            - name: Download latest Simple Icons release
+              run: |
+                Invoke-WebRequest -Uri "https://github.com/simple-icons/simple-icons/archive/refs/tags/${{ needs.version.outputs.LATEST_TAG_NAME }}.zip" -OutFile "simple-icons-${{ needs.version.outputs.LATEST_TAG_NAME }}.zip"
+              shell: pwsh
+
+            - name: Extract Simple Icons
+              run: 7z x simple-icons-${{ needs.version.outputs.LATEST_TAG_NAME }}.zip -o./simple-icons-${{ needs.version.outputs.LATEST_TAG_NAME }}
+              shell: cmd
+
+            - name: Install python packages
+              run: |
+                python -m pip install --upgrade pip
+                pip install -r requirements.txt
+
+            - name: Run script/main.py with the latest Simple Icons release
+              run: |
+                cd script
+                python main.py ./../simple-icons-${{ needs.version.outputs.LATEST_TAG_NAME }}/simple-icons-${{ needs.version.outputs.LATEST_TAG_NAME }} ${{ needs.version.outputs.LATEST_TAG_NAME }}
+                cd ..
```

The working directory is not preserved between steps. Also, you can use [`working-directory`](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsworking-directory):

```suggestion
              working-directory: scripts
              run: python main.py ./../simple-icons-${{ needs.version.outputs.LATEST_TAG_NAME }}/simple-icons-${{ needs.version.outputs.LATEST_TAG_NAME }} ${{ needs.version.outputs.LATEST_TAG_NAME }}
```